src/buy-api.js

```diff
@@ -49,7 +49,7 @@ const getItemByItemGroup = function (itemGroupId) {
 
 const searchItems = function (searchConfig) {
     if (!searchConfig) throw new Error('Error --> Missing or invalid input parameter to search');
-    if (!searchConfig.keyword && !searchConfig.categoryId && !searchConfig.gtin) throw new Error('Error --> Keyword or category id is required in query param');
+    if (!searchConfig.keyword && !searchConfig.categoryId && !searchConfig.gtin && !searchConfig.epid) throw new Error('Error --> Keyword or category id is required in query param');
     if (!this.options.appAccessToken) throw new Error('Error -->Missing Access token, Generate access token');
```

Comment:
epid is not required field, you can remove this from the if loop.

https://developer.ebay.com/api-docs/buy/browse/resources/item_summary/methods/search#h2-input

Reply:
Is not possible to seach only by epid then?

Reply:
@rambonette You are right, we can search using epid. We can change the error message like this 

`'Error --> Kindly provide the valid Keyword, category id, epid or gtin in query param`

Reply:
@pajaydev That's perfect! I readded the `!searchConfig.epid` condition and updated the error message with the text you provided! Let me know if there's anything more to be changed :)

Reply:
@rambonette Cool. Thanks I appreciate it.